hw/ip/dma/data/dma.hjson

```diff
@@ -50,24 +69,41 @@
       hwre:     "true", // enable `re` latched signal of software read pulse
       resval:   1,
       fields: [
-        { bits: "0", name: "READY", desc: "Transaction iss done"},
+        { bits: "0", name: "READY", desc: "Transaction is done"},
         { bits: "1", name: "WINDOW_DONE", desc: "set if DMA is copying second half"},
       ]
     },
-    { name:     "PTR_INC",
+    { name:     "SRC_PTR_INC",
       desc:     "Increment number of src/dst pointer every time a word is copied",
       swaccess: "rw",
       hwaccess: "hro",
       fields: [
         { bits: "7:0", 
-          name: "SRC_PTR_INC", 
-          desc: "Source pointer increment",
+          name: "SRC_PTR_INC_D1", 
```

Comment:
why one pointer increases by max 256 (8b) while the one below by 16b?

Reply:
Good call. I modified both of them to support a stride of 16 words on both dimensions, so:
- src_ptr_inc_d1 on 6 bits: 64B
- src_ptr_inc_d2 on x bits: 
This is tricky, because to avoid computing multiplitcations this register holds the number of bytes to increment the read_ptr to go to the next row*stride. 
For this reason, this register has to hold, in the worst case scenario:
biggest possible row (or first dimension): 64kB + (64kB - 1) * 64B = 532 MB -> 23 bits

Reply:
let's discuss about it in a meeting cause it is not clear to me